### PR TITLE
Add ai2z to testing

### PR DESCRIPTION
**Pull request by SebastianGuadalupe**

### CODE REVIEW

File: test/identity_network.jl [hunk 1]
```diff
@@ -30,6 +30,22 @@
 
     end
 
+    @testset "Group 1.1" begin
```
Comment (mforets) [inline test/identity_network.jl:33]: is there any special reason to have this in its own separate group?

Comment (SebastianGuadalupe) [reply test/identity_network.jl:33]: It's because the algorithm cannot use an `HPolytope` as an input

Comment (mforets) [reply test/identity_network.jl:34]: ok, i see. seems like a limitation that should be lifted. either by converting or overapproximating to a zonotope. in fact we can dispatch on zonotopic sets (ie. AbstractZonotope) and the implementation of LazySets should still hold. this covers hyperrectangles as a particular case.

https://github.com/JuliaReach/NeuralVerification.jl/blob/4a7cb4f941504ef4312bc59cddc0797f14e6c4ed/src/reachability/ai2z.jl#L34

Comment (mforets) [reply test/identity_network.jl:34]: for general HPolytope's i would suggest to take an overapproximation, say a box as a first step.

**Comment by mforets** [comment]
Nice!

**Pull request merged**